twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -251,16 +251,7 @@ def _compile_scala(
 
 def _empty_java_info(deps_java_info, implicit_deps):
```

Comment:
can we remove this entire method? it seems like it is just `java_common.merge(deps_java_info + implicit_deps)` now, and I'd rather that be inlined.

Reply:
Sure, done.